### DISCUSSION TITLE
Looking for Customized Demo code

### DISCUSSION THREAD

**Discussion by catative** [scottyzen/woonuxt#264] [General]
@scottyzen Hello, thank you very much for your work. Could you tell me please where I can find the source code of this project https://myshop.woonuxt.com/ for an example? I can't figure out how to develop my own online store based on your code

**Comment by fabdao** [comment]
Hi @catative What's up ?

 [Here my  boilersplate for woonuxt and docker](https://github.com/fabdao/docker-woonuxt)

I try my best with the readme me. It's explain all you need to install. 

Best regard !
